src/javascript/Create/Create.container.jsx

```diff
@@ -14,22 +14,22 @@ const mapDispatchToProps = dispatch => ({
 });
 
 const mapStateToProps = state => {
-    const contentEditorUiLang = Constants.supportedLocales.includes(state.uiLang) ?
-        state.uiLang :
+    const contentEditorUiLang = Constants.supportedLocales.includes(state.uilang) ?
```

Comment:
Why doing this kind of change ?

Reply:
Because it needed to be inline with CMM and in CMM it was a mix of uiLang and uilang until I made it all lowercase. We can go whichever way we want I guess (lower or upper), but I believe there needs to be some kind of consistency between projects especially if we share constants like that. Also in jahia-private we have `public static final String UI_LANGUAGE = "uilang";` I happened to be working in all of these projects recently and the inconsistency drove me nuts.